actions/ostree_commit_action.go

```diff
@@ -75,8 +117,22 @@ func (ot *OstreeCommitAction) Run(context *debos.DebosContext) error {
 		return err
 	}
 
+	if err = ot.parseMeta(context); err != nil {
+		return err
+	}
+
 	opts := otbuiltin.NewCommitOptions()
 	opts.Subject = ot.Subject
+	for k, v := range ot.meta {
+		str := fmt.Sprintf("%s=%s", k, v)
+		opts.AddMetadataString = append(opts.AddMetadataString, str)
+	}
+
+	if ot.CollectionID != "" {
+		opts.CollectionID = ot.CollectionID
+		opts.RefBindings = append(opts.RefBindings, ot.Branch)
```

Comment:
refbindings should be configurable as extra brances to add on top of the current one. e.g. for being able to switch between a development and production branch.

Also is it correct to do reference binding *only* when there is a collection id and if there is one then *enforce* it ? that seems like encoding a lot of policy.

Reply:
From my findings "ref-binding" should be inserted in case if "collection-id" added.

Add following logic:
If "ref-bindings:" is absent but "collection-id:" is set, then implicitly add only current branch.  
if "ref-bindings:" parameter is set explicitly for "ostree-commit" action, then set only listed branches from that parameter despite of "collection-id".